index.js

```diff
@@ -0,0 +1,155 @@
+    name: 'FACEBOOK',
+    back: 'Full Stack Dev',
+    SourceLink: 'https://github.com/jerymey',
+    LiveLink: 'https://jerymey.github.io/My-portfolio/',
+    years: '2015',
+    Image: 'images/tonic.png',
+    descrip: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+  },
+  {
+    id: 3,
+    title: 'Facebook 360',
+    name: 'FACEBOOK',
+    back: 'Full Stack Dev',
+    SourceLink: 'https://github.com/jerymey',
+    LiveLink: 'https://jerymey.github.io/My-portfolio/',
+    years: '2015',
+    Image: 'images/yoga.png',
+    descrip: "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+
+  },
+  {
+    id: 4,
+    title: 'Uber Navigation',
+    name: 'Uber',
+    back: 'Lead Developer',
+    SourceLink: '[@githubhandle](https://github.com/jerymey)',
+    LiveLink: 'https://jerymey.github.io/My-portfolio/',
+    years: '2018',
+    Image: 'images/yoga.png',
+    descrip: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+  },
+];
+document.getElementById('work').innerHTML = projects.map((item) => `  
+    <li class="works__card flex bg-white">
+    <div class="works__card__img">
+      <img src="${item.Image}" alt="professional art printing project" />
+    </div>
+    <div class="works__card__text">
+      <h2 class="works__card__title fw-700 fz-32 lh-44 ctr-200">
+       ${item.title}
+      </h2>
+      <div class="works__card__subtitle flex">
+        <p class="works__card__subtitle-1 fw-600 fz-13 lh-16 clr-n-600">
+         ${item.name}
+        </p>
+        <ul class="works__card__subtitle-ul flex">
+          <li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">
+           ${item.back}
+          </li>
+          <li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">
+          ${item.years}
+          </li>
+        </ul>
+      </div>
+      <p class="works__card__dsc fw-400 fz-15 lh-24 clr-n-600">
+       ${item.descrip}
+      </p>
+      <div class="works__card__topics flex">
+        <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+          html
+        </p>
+        <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+          Ruby on rails
+        </p>
+        <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+          css
+        </p>
+        <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+          javascript
+        </p>
+      </div>
+      <div class="works__card__btn">
+        <button type="button" class="btn fw-500 fz-17 lh-24 clr-p-500 bg-white" onclick="display(${item.id})">
+          See Project
+        </button>
+      </div>
+    </div>
+  </li>
+    <div class="popup-window" id="element-${item.id}">
```

Comment:
- [ ] The purpose of this milestone is to have the popup cards  match the given figma design, both for mobile and for the desktop.

Your Mobile design
![image](https://github.com/jerymey/My-portfolio/assets/81932478/9ceaf65f-0537-43ae-98db-342df16e864f)

Given Design
![image](https://github.com/jerymey/My-portfolio/assets/81932478/8028daca-49e6-45f1-95c4-7d3982867342)

- The close icon should be aligned properly to the right, and I don't know if its a machine thing on my end but the icon is a word
- The circles in the part below should be full grey dots, this can be achieved by making them `divs`, giving them a 59% radius so they become circles, a bit of width and height e.g. 12% so they are visible
![image](https://github.com/jerymey/My-portfolio/assets/81932478/6864ca11-267e-4f7a-bd68-db55cc1545af)
- The color of the text below the image should match the given design
- The languages and the buttons should follow the layout of the given design
- The buttons should also follow the given color which is white with a blue border and blue icons
![image](https://github.com/jerymey/My-portfolio/assets/81932478/bde8a134-eef4-45f1-aa73-5fb46121392c)


### Desktop View
Your Desktop View
![image](https://github.com/jerymey/My-portfolio/assets/81932478/0e228f23-5bdf-49e4-8522-987a658595e0)

Given Design
![image](https://github.com/jerymey/My-portfolio/assets/81932478/58a8fd4f-e047-4ae4-a753-78848318c37a)

- The close icon should aligned to the left
- The size of the image should match the size of the image in the given design
- The text is aligned to the left and the languages and the buttons should be aligned to the right as is in the given design.
#### Note that padding and margin should be applied accordingly to achieve the nice design we have been given to follow